src/main/resources/avro/common.avdl

```diff
@@ -128,9 +130,9 @@ record CigarUnit {
   /**
   `referenceSequence` is only used at mismatches (`SEQUENCE_MISMATCH`)
   and deletions (`DELETE`). Filling this field replaces the MD tag.
-  If the relevant information is not available, leave this field as `null`.
+  If the relevant information is not available, leave this field empty.
   */
-  union { null, string } referenceSequence = null;
+  string referenceSequence;
```

Comment:
I guess I'm not clear on how these two cases are differentiable in Avro, so I defaulted to the syntax which *looks* closest to proto3. Isn't a default of `""` assumed if my type is `string`?

`string x = "";` vs `string x;`

That said, if the former makes things easier for the reference server or whatever Avro parsing scripts we have, I'll gladly make the change.